test/library/standard/Random/RandomStreamInterface/choiceTestUtils.chpl

```diff
@@ -48,20 +52,20 @@ proc test(stream, X, size:?sizeType=none, replace=true, prob:?probType=none, tri
   var m = 1;
   if isDomainType(sizeType) then m = size.size;
   else if isIntegralType(sizeType) then m = size;
-  
+
   var actualRatios = new map(int, real);
   for (k,v) in actualRatios.items() {
     if isNothingType(sizeType) then
       actualRatios[k] = v / trials:real;
     else
       actualRatios[k] = v / (trials*m): real;
   }
-  
+
   var dom: domain(1) = {1..X.size};
   var ones: [dom] real = 1;
 
   var probabilities = if isNothingType(prob.type) then ones
-                      else prob.reindex(1..X.size);
+                      else reshape(prob, {1..X.size});
```

Comment:
I am seeing a similar change here, so there must be something that I am missing that makes this necessary.